Documentation/Contributing.md

```diff
@@ -0,0 +1,160 @@
+# Swift Package Manager: Quick Start
+Swift Package Manager (SwiftPM) is a tool for building, testing and managing Swift project dependencies.  
+In order to use it you will need Swift 3.0 or greater.  
+SwiftPM is also included in Xcode 8.0 and above.  
+For usage getting started: [https://swift.org/getting-started/#using-the-package-manager](https://swift.org/getting-started/#using-the-package-manager)  
+For overview and examples: [https://swift.org/package-manager](https://swift.org/package-manager/)
```

Comment:
This paragraph is redundant in this context, I would start this  document with the text under "Code Contributions" section